chunky/src/java/se/llbit/chunky/resources/texturepack/EntityTextureLoader.java

```diff
@@ -32,10 +33,18 @@
 public class EntityTextureLoader extends TextureLoader {
```

Comment:
So this "entity" texture loader actually only loads player textures? Should it be renamed then?

Reply:
Its also used [here](https://github.com/chunky-dev/chunky/blob/master/chunky/src/java/se/llbit/chunky/resources/TexturePackLoader.java#L2466) for a few other mobs but I don't think it actually does anything?

Reply:
Oh, okay then. The `SimpleTexture` assumes that a non-animated texture is a square so that might be why a different loader is needed. It can stay this way then. 👍

Reply:
Only the heads of those mobs are used so that's probably why it works.